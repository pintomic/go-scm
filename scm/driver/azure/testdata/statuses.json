[
    {
        "Id": 1,
        "State": "succeeded",
        "Links": "",
        "Description": "test-repo build succeeded",
        "TargetUrl": "vstfs:///Build/Build/12345",
        "Context": {
            "Name": "build/test-repo",
            "Genre": "continuous-integration"
        }
    }
]