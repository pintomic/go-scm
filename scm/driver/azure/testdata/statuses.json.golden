[
    {
        "State": "success",
        "Label": "build/test-repo",
        "Desc": "test-repo build succeeded",
        "Target": "vstfs:///Build/Build/12345"
    }
]